clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/pro-bounds-avoid-unchecked-container-accesses.rst

```diff
@@ -0,0 +1,51 @@
+.. title:: clang-tidy - cppcoreguidelines-pro-bounds-avoid-unchecked-container-accesses
+
+cppcoreguidelines-pro-bounds-avoid-unchecked-container-accesses
+===============================================================
+
+Finds calls to ``operator[]`` in STL containers and suggests replacing them
+with safe alternatives.
```

Comment:
I'd prefer it if the docs just mention shortly by name what possible alternatives there are, e.g., 
> `span`, `at()`, `gsl::at`, `range-for`, `begin(),end()` or algorithms 

(maybe not in the first sentence to keep the sync work low, but as a follow up/explanation.)

Reply:
What do you think?

```diff
 Finds calls to ``operator[]`` in STL containers and suggests replacing them
 with safe alternatives.
+Safe alternatives include STL ``at`` or GSL ``at`` functions, ``begin()`` or
+``end()`` functions, ``range-for`` loops, ``std::span``, or an appropriate
+function from ``<algorithms>``.
```

Reply:
That sounds good, thanks